samples/subsys/mgmt/mcumgr/smp_svr/sample.yaml

```diff
@@ -102,3 +102,28 @@ tests:
       - mg100
     integration_platforms:
       - nrf52840dk/nrf52840
+  sample.mcumgr.smp_svr.ram_load:
+    sysbuild: true
+    extra_args: FILE_SUFFIX="ram_load"
+    platform_allow:
+      - nrf52840dk/nrf52840
+    integration_platforms:
+      - nrf52840dk/nrf52840
+  sample.mcumgr.smp_svr.ram_load.serial:
+    sysbuild: true
+    extra_args: FILE_SUFFIX="ram_load"
+                OVERLAY_CONFIG="overlay-serial.conf"
+    platform_allow:
+      - nrf52840dk/nrf52840
+    integration_platforms:
+      - nrf52840dk/nrf52840
+  sample.mcumgr.smp_svr.ram_load.serial.fs.shell:
+    sysbuild: true
+    extra_args: FILE_SUFFIX="ram_load"
+                OVERLAY_CONFIG="overlay-serial.conf"
```

Comment:
OVERLAY_CONFIG is obsolete. since v3.4 https://docs.zephyrproject.org/latest/releases/release-notes-3.4.html#build-system-and-infrastructure
**EXTRA_CONF_FILE replaces OVERLAY_CONFIG.**

Reply:
Fixed